src/main/java/org/imsglobal/caliper/events/ResourceManagementEvent.java

```diff
@@ -60,16 +59,16 @@ public static abstract class Builder<T extends Builder<T>> extends Event.Builder
          * Constructor
          */
         public Builder() {
-            super.type(EventType.CONTENT);
+            super.type(EventType.RESOURCE_MANAGEMENT_EVENT);
         }
 
         /**
          * Client invokes build method in order to create an immutable profile object.
-         * 
-         * @return a new ContentEvent instance.
+         *
+         * @return a new PathEvent instance.
```

Comment:
Looks like the comment is wrong

Reply:
Right, this should be ResourceManagementEvent. Correcting this.